new-pipeline/dist.html

```diff
@@ -55,7 +55,10 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                 </h2>
                 <hr></hr>
                 <figure class="col-md-12">
-                  <figcaption id="dist-caption"></figcaption>
+	                <div class="caption-div">
+	                  <figcaption id="dist-caption-text" class="overflow ellipsis"></figcaption>
+	                  <figcaption id="dist-caption-link"></figcaption>
+	                </div>
                     <div class="col-md-9" id="plots">
```

Comment:
The indentation is still a bit off here.
The goal would be to have `dist-caption` on the same indentation level as `plots` (as they are at the same level in the hierarchy here).

Reply:
@georgf , I put `dist-caption` on the same level as that of `plots` and again commited. But it still shows it like this, they are on the same level in my code on my local machine. Can you please tell the probable reasons. Thanks!

Reply:
Are you mixing tabs and spaces here?
Most code editors have an option to show these with extra characters.
In the code in this repository, we only use spaces for indentation, not tabs.

Reply:
@georgf , This is how the code looks like in my editor. 

![screenshot from 2018-03-10 13-17-39](https://user-images.githubusercontent.com/23060217/37239633-70bd9de8-2465-11e8-9603-e54a7758250d.png)

Please have a look. Thanks!

Reply:
If you look in the ["files changed" view](https://github.com/mozilla/telemetry-dashboard/pull/418/files) you can see that it looks different.

To fix this, you can check for an option to show tabs & spaces in your editor, so you can see this.
More generally you can check for an option in your editor that let's you choose to use spaces instead of tabs for indentation.

Reply:
@georgf , Yeah I can see the difference when I push the branch on GitHub. But it is not visible in my editor. I already have `Indent using spaces` setting in my editor. I will try again and hope to fix it. Thanks!